disnake/http.py

```diff
@@ -933,6 +917,16 @@ def edit_voice_state(
         )
         return self.request(r, json=payload)
 
+    def edit_my_member(
+        self,
+        guild_id: Snowflake,
+        *,
+        reason: Optional[str] = None,
+        **fields: Any,
```

Comment:
Any reason not to explicitly specify allowed parameters here? The method returns a `member.Nickname` dict, I feel like either the parameters should be specified or the return type should be made more generic

Reply:
I based the method definition here off of `edit_member` (below), which uses `**fields: Any` with no specific validation.

Reply:
7bba3032c873514542727845f9815305b91b1b8a